SlackAPI/SlackClient.cs

```
@@ -645,5 +645,16 @@ public void UploadFile(Action<FileUploadResponse> callback, byte[] fileData, str
                 callback(result.Deserialize<FileUploadResponse>());
             }
         }
+
+        public void DeleteFile(Action<FileDeleteResponse> callback, string file = null)
+        {
+            List<Tuple<string, string>> parameters = new List<Tuple<string, string>>();
+
+            if (!string.IsNullOrEmpty(file))
```

Should probably just throw an exception or return early (before List creation)